README.md

```diff
@@ -1 +1,72 @@
-# usePolling
+# use-raf-polling
+
+## Overview
+
+*use-raf-polling* is a lightweight npm package that provides a simple and intuitive hook for API polling in React applications. The package simplifies the process of retrieving the fresh data in your applications, making them more responsive and engaging for your users.
+
+## Features and benefits
+
+- Easy to use: *use-raf-polling* provides a straightforward way to implement polling in your React components, with no complex configuration required.
+- Flexible: you can customize the polling interval, polling condition, callback functions to suit your specific use case.
+- Efficient: *use-raf-polling* uses requestAnimationFrame to ensure smooth and performant polling, without blocking the main thread.
```

Comment:
how about adding/mentioning that callback will not be executed while browser tab is not active?

Reply:
added